### DISCUSSION TITLE
Files in Third Party Exercises

### DISCUSSION THREAD

**Discussion by cshokie87** [rust-lang/rustlings#2170] [Q&A] [answered]
I am trying to use rustlings third party framework to create exercises for some common activities, such as file io. This lead to trying to have test files to read from and write to in the test directory. When I runrustlings dev check I get the following error: 

Updated `Cargo.toml`
Running all exercises to check that they aren't already solved...
Progress: 0/2Error: Found the file `exercises/01_fileio/test_ascii.txt`. Only `README.md` and Rust files related to an exercise in `info.toml` are allowed in the `exercises` directory. 

How do I annotate info.toml to say this file is expected and part of the test?

**Comment by georgeeburt** [comment]


**Reply by mo8it** [reply]
AI bullshit hallucination… Your account is reported.

**Comment by mo8it** [comment, marked as answer]
@cshokie87 Rustlings doesn't support additional files yet. I will add this support in the next version :)

**Reply by cshokie87** [reply]
Thanks for getting back to me. I look forward to the next release. 

It's a shame, the AI was relatively convincing and appeared on point.
